.github/workflows/build.yaml

```diff
@@ -20,9 +20,9 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v3
       with:
-        go-version: 1.18
+        go-version: '^1.20.x'
```

Comment:
Since it's already required in [go.mod](https://github.com/lcd1232/go-appimage/blob/42e24a35e10eb0c661e18f565c734a3555ce9665/go.mod#L3-L2)